### ISSUE TITLE
Numeric fields must be strings

### ISSUE THREAD

**Issue by Yehonal** [sequelize/sequelize-auto#336]
Using sequelize-auto on a table with a field that starts with a number it creates a model with an object property that is not allowed by javascript syntax.

For example i've a fields in mysql called: **1_phone, 2_phone etc.**
i cannot change the mysql field itself so sequelize-auto should be able to create the model using string as property name instead:

```
define("mytable",{
    ...
    "1_phone" : { ... },
    "2_phone" : { ... },
}
```

I could do it manually but i would avoid to touch models and keep them updated with sequelize-auto instead.

**Comment by aruntk** [comment]
I have fixed this in @tkay/sequelize-auto. Until they accept the PR you can use this.

**Comment by steveschmitt** [comment]
Fixed in  `sequelize-auto@0.5.1`.
